package.json

```diff
@@ -45,5 +45,13 @@
     "router",
     "route-recognizer",
     "rsvp"
-  ]
+  ],
+  "dependencies": {
```

Comment:
The issue is, if one wants to use routerjs outside of Ember (which I do, for virtually all of my frontend dev projects), one is unlikely to have route-recognizer or rsvp already installed. It seems then that these are logically dependencies, not peerDependencies. By specifying that these are both dependencies and peerDependencies, we essentially wind up installing rsvp and route-recognizer if they are not already present.

Alternatively, I can drop the dependencies and just add a line to the README about how to install via NPM.